tensorrt_llm/_torch/auto_deploy/custom_ops/linear/linear.py

```
@@ -19,6 +19,21 @@
+from tensorrt_llm._utils import get_sm_version
+
+# Cache sm version (call once).
+_SM_VERSION: Optional[int] = None
+
+
+def _sm_version() -> int:
+    global _SM_VERSION
+    if _SM_VERSION is None:
+        try:
+            _SM_VERSION = get_sm_version()
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)